packages/ripple-nuxt-tide/README.md

```diff
@@ -67,7 +244,133 @@ You will need below config file in your Nuxt root to make @dpc-sdp/ripple-nuxt-t
 
 ### Custom theming
 
-TODO...
+Theme customisations are possible and allow you to modify the default style of
+your site.
+
+Customisations to the theme should be added to the following root directories:
+
+- `/assets` - Custom [theme](#Theme-variable-overrides), [stylesheets](#Custom-stylesheets), [icons](#Custom-icons).
+- `/static` - Custom [images](#Custom-images) (favicons, file icons, header and error graphics).
+
+#### Theme variable overrides
```

Comment:
I feel this part should be in our ripple `components` doc. Not `ripple-nuxt-tide`.

Reply:
Currently there isn't any good place within components to place this. Components have their own generated README.md files (which we could remove if needed?) Perhaps ripple-global? I need some direction for this.

Reply:
I would like to be able to add usage info to a ripple component README.md, which I dont think we can do currently. Perhaps we could add a USAGE.md file which could get injected into the README.md with the generate docs script? Or otherwise just add a link in README.md if USAGE.md exists?